### PR TITLE
Various item_mods fixes

### PR DESCRIPTION
**Pull request by wrenffxi**

### CODE REVIEW

File: sql/item_mods.sql [hunk 1]
```diff
@@ -2683,44 +2685,45 @@ INSERT INTO `item_mods` VALUES (10906,24,6);
...
+INSERT INTO `item_mods` VALUES (10910,28,4);      -- Magic Atk. Bonus +4
+INSERT INTO `item_mods` VALUES (10910,170,7);     -- Fast Cast +7%
+INSERT INTO `item_mods` VALUES (10910,296,4);     -- Conserve MP +4
+INSERT INTO `item_mods` VALUES (10910,562,4);     -- Magic critical hit rate +4%
+INSERT INTO `item_mods` VALUES (10911,1,29);      -- Nares Cap: DEF 29
+INSERT INTO `item_mods` VALUES (10911,12,12);     -- INT +12
+INSERT INTO `item_mods` VALUES (10911,28,5);      -- Magic Atk. Bonus +5
+INSERT INTO `item_mods` VALUES (10911,170,10);    -- Fast Cast +10%
+INSERT INTO `item_mods` VALUES (10911,296,5);     -- Conserve MP +5
+INSERT INTO `item_mods` VALUES (10910,562,5);     -- Magic critical hit rate +5%
```
Comment (lichtl) [inline sql/item_mods.sql]: I think this should be 10911 otherwise its a duplicate of line 2706

Comment (wrenffxi) [inline sql/item_mods.sql]: Fixed. Thanks!

File: sql/item_mods.sql [hunk 2]
```diff
@@ -37387,8 +37386,6 @@ INSERT INTO `item_mods` VALUES (28665,29,6);
 INSERT INTO `item_mods` VALUES (28665,109,106);
 INSERT INTO `item_mods` VALUES (28665,161,-8);
 INSERT INTO `item_mods` VALUES (28670,1,1);
-INSERT INTO `item_mods` VALUES (28757,23,14);
```
Comment (TeoTwawki) [inline sql/item_mods.sql]: I think the original author meant 28575
http://www.ffxiah.com/item/28575/choj-band

Comment (wrenffxi) [inline sql/item_mods.sql]: You're probably right.  There's already an entry for 28575; no need to add it.

**Comment by wrenffxi** [comment]
Hm, I think I messed something up there.

**Comment by teschnei** [comment]
rebase your commits on top of everything you just pulled

**Comment by takhlaq** [comment]
and force push

**Comment by takhlaq** [comment]
```
Importing item_mods.sql into the database...ERROR 1062 (23000) at line 24597: Duplicate entry '26628-519' for key 'PRIMARY'
```

**Comment by TeoTwawki** [comment]
You realize the more you add onto the open pr, the longer it takes to get reviewed right?

**Comment by wrenffxi** [comment]
Am open to suggestions.  You would rather each commit have a separate PR?  Or should I stop making adjustments to item_mods for a while, to let you all catch up on review?

**Comment by TeoTwawki** [comment]
I'd suggest letting them catch up on whats here and merge it, then make more after. You have a healthy chunk of changes already (which we're grateful for).

**Comment by takhlaq** [comment]
thanks for keeping your commits small enough to review :)

**Pull request merged**